DICOM/src/Microsoft.Health.Anonymizer.Common/Settings/RedactSetting.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Anonymizer.Common.Settings
+{
+    public class RedactSetting
+    {
+        public bool EnablePartialDatesForRedact { get; set; } = false;
+
+        public bool EnablePartialZipCodesForRedact { get; set; } = false;
+
+        public bool EnablePartialAgesForRedact { get; set; } = false;
+
+        public List<string> RestrictedZipCodeTabulationAreas { get; set; }
```

Comment:
we can have a default empty list here so that we don't need to handle meaningless null values

Reply:
👍 That’s the same reason I comment setting the key in encryptionSetting. It is not good to initialize an object with only some “easy” fields. But unfortunately we have to handle the null values because the user can create an object and set this field to null…

Reply:
Agree. We can make the list fields readonly.